diff --git a/node_modules/react-native/scripts/cocoapods/flipper.rb b/node_modules/react-native/scripts/cocoapods/flipper.rb
index e2bfe76..1e9416f 100644
--- a/node_modules/react-native/scripts/cocoapods/flipper.rb
+++ b/node_modules/react-native/scripts/cocoapods/flipper.rb
@@ -52,7 +52,7 @@ def use_flipper_pods(versions = {}, configurations: ['Debug'])
     pod 'Flipper-Boost-iOSX', versions['Flipper-Boost-iOSX'], :configurations => configurations
     pod 'Flipper-DoubleConversion', versions['Flipper-DoubleConversion'], :configurations => configurations
     pod 'Flipper-Fmt', versions['Flipper-Fmt'], :configurations => configurations
-    pod 'Flipper-Folly', versions['Flipper-Folly'], :configurations => configurations
+    pod 'Flipper-Folly', :podspec => "../Flipper-Folly.podspec", :configurations => configurations
     pod 'Flipper-Glog', versions['Flipper-Glog'], :configurations => configurations
     pod 'Flipper-PeerTalk', versions['Flipper-PeerTalk'], :configurations => configurations
     pod 'Flipper-RSocket', versions['Flipper-RSocket'], :configurations => configurations
@@ -64,7 +64,7 @@ def use_flipper_pods(versions = {}, configurations: ['Debug'])
     pod 'FlipperKit/FlipperKitHighlightOverlay', versions['Flipper'], :configurations => configurations
     pod 'FlipperKit/FlipperKitLayoutTextSearchable', versions['Flipper'], :configurations => configurations
     pod 'FlipperKit/FlipperKitNetworkPlugin', versions['Flipper'], :configurations => configurations
-    pod 'OpenSSL-Universal', versions['OpenSSL-Universal'], :configurations => configurations
+    pod 'OpenSSL-Universal', :git => 'https://github.com/krzyzanowskim/OpenSSL.git', :tag => "1.0.220", :configurations => configurations
 end

 # Applies some changes to some pods of the project:
